### PR TITLE
Add changes to support CFT AKS Terraform changes

### PR DESCRIPTION
**Pull request by thomast1906**
### Change description ###
Add changes to support CFT AKS Terraform changes 
- Have updated terraform.state files

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by thomast1906** [comment]
Do not merge for now added - want to view TF Plans

File: environments/sbox/sbox.tfvars [hunk 1]
```diff
@@ -2,18 +2,18 @@ project                    = "hmcts"
 location                   = "uksouth"
 env                        = "sbox"
 subscription               = "sbox"
-certificate_key_vault_name = "cftapps-sbox"
+certificate_key_vault_name = "dcdcftappssboxkv"
```
Comment (timja) [inline environments/sbox/sbox.tfvars:5]: should use `acmedcdcftappssbox` instead

File: components/cftapps_cluster_lb_backend/local.tf [hunk 2]
```diff
@@ -1,8 +1,8 @@
 locals {
   env = (var.env == "aat") ? "stg" : "${(var.env == "perftest") ? "test" : "${var.env}"}"
 
-  vnet_rg   = "aks-infra-${var.env}-rg"
-  vnet_name = "core-${var.env}-vnet"
+  vnet_rg   = var.env == "sbox" ? "cft-${var.env}-network-rg" : "aks-infra-${var.env}-rg"
+  vnet_name = var.env == "sbox" ? "cft-${var.env}-vnet" : "core-${var.env}-vnet"
 
-  key_vault_resource_group = var.env == "perftest" || var.env == "aat" ? "core-infra-${var.subscription}-rg" : "core-infra-${var.env}-rg"
+  key_vault_resource_group = var.env == "sbox" ? "genesis-rg" : var.env == "perftest" || var.env == "aat" ? "core-infra-${var.subscription}-rg" : "core-infra-${var.env}-rg"
```
Comment (timja) [inline components/cftapps_cluster_lb_backend/local.tf:7]:
```suggestion
  key_vault_resource_group = var.env == "sbox" ? "cft-platform-${var.env}-rg" : var.env == "perftest" || var.env == "aat" ? "core-infra-${var.subscription}-rg" : "core-infra-${var.env}-rg"
```

**Review by timja — APPROVED** [review]

**Review by timja — APPROVED** [review]

File: environments/sbox/sbox.tfvars (hunk 1 above)
Comment (thomast1906) [reply environments/sbox/sbox.tfvars]: Updated

**Review by timja — APPROVED** [review]

**Pull request merged**